<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LTCH-5CD105HTG9" name="restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations" tests="11" failures="2" timestamp="9 Jun 2021 08:12:52 GMT" time="184.410" errors="0">
  <testcase name="MethodologyItemRelation_PostCreateANewMethodologyRelation_status400" time="21.711" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations"/>
  <testcase name="MethodologyItemRelation_DeleteLinkedMethodologyItem_status400" time="26.233" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations"/>
  <testcase name="MethodologyItemRelation_ListOfAllMethodologyRelation_status400" time="4.504" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations">
    <failure type="java.lang.AssertionError" message="expected [400] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [200]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations.MethodologyItemRelation_ListOfAllMethodologyRelation_status400(Restassured_Automation_MethodologyItemRelations.java:690)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- MethodologyItemRelation_ListOfAllMethodologyRelation_status400 -->
  <testcase name="MethodologyItemRelation_UpdateAMethodologyRelation_status400" time="22.915" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations"/>
  <testcase name="MethodologyItemRelation_MethodologyItemRelation_EndToEnd_Scenario" time="12.771" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations"/>
  <testcase name="MethodologyItemRelation_PostCreateANewMethodologyRelation_status200" time="21.422" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations"/>
  <testcase name="MethodologyItemRelation_ListOfAllMethodologyRelation_status200" time="4.197" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations"/>
  <testcase name="MethodologyItemRelation_DeleteLinkedMethodologyItem_status200" time="26.401" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations"/>
  <testcase name="MethodologyItemRelation_UpdateAMethodologyRelation_status200" time="22.552" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations"/>
  <testcase name="MethodologyItemRelation_DeleteAnMethodologyRelation_status204" time="9.623" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations"/>
  <testcase name="MethodologyItemRelation_DeleteAnMethodologyRelation_status400" time="12.081" classname="restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations">
    <failure type="java.lang.AssertionError" message="expected [400] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [500]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations.MethodologyItemRelation_DeleteAnMethodologyRelation_status400(Restassured_Automation_MethodologyItemRelations.java:1493)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- MethodologyItemRelation_DeleteAnMethodologyRelation_status400 -->
</testsuite> <!-- restassured.automation.testcases.Restassured_Automation_MethodologyItemRelations -->
